src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -361,6 +361,15 @@ class MacroAssembler: public Assembler {
                                        uint32_t& predecessor, uint32_t& successor) {
```

Comment:
Should we also move membar_mask_to_pred_succ into fence?

Reply:
No. That change is to match what's upstream again. See https://github.com/openjdk/jdk/compare/master...rivosinc:jdk:rivos/main#diff-0b26e6e1a71a439a624c92b9bcccf985eef787d36b8825a3a53cf308b75e6ca0L364-L372